www/js/home.js

```diff
@@ -17,79 +17,132 @@ function loadHome() {
 }
 
 function initHome() {
-  var newsTab = $('#subtab-news');
-  var eventTab = $('#subtab-event');
+  let newsTab = $('#subtab-news');
+  let eventTab = $('#subtab-event');
```

Comment:
Use `const` instead of `let` for values that are not reassigned.

Reply:
Forgot to fix this...